user-registration-app/src/app/models/registration.ts

```diff
@@ -128,3 +96,82 @@ export interface IRegistrationDb {
   base_pin: string;
   active: boolean;
 }
+
+export class RegistrationApiResponse {
+  academic_year: string;
+  active: boolean;
+  allergies: string | null;
+  base_pin: string;
+  coding_experience: string | null;
+  dietary_restriction: string | null;
+  eighteenBeforeEvent: boolean;
+  email: string;
+  end_time: string;
+  expectations: string | null;
+  first_hackathon: boolean;
+  firstname: string;
+  gender: string;
+  lastname: string;
+  major: string;
+  mlh_coc: boolean;
+  mlh_dcp: boolean;
+  name: string;
```

Comment:
Is 'name' used somewhere else in the code? Just wondering if it's needed since we already have 'firstname' and 'lastname'

Reply:
`name` is used by Hackathon.parseJSON()`. It refers to the name of the hackathon which shows up on the profile page.